### ISSUE TITLE
Remove low-level EVM API from Client

### ISSUE THREAD

**Issue by enlight** [loomnetwork/loom-js#54]
The public `Client` API has become somewhat bloated with methods that are only used by Loom Provider, `Client.getEvmLogsAsync()` for example returns a single `Uint8Array` which end users wouldn't have a clue how to convert into actual log entries (nor should they).

**Comment by eduardonunesp** [comment]
@enlight , Yep those functions are very low level, maybe `99%` of users will prefer to use `web3` or call `JSON RPC` to `LoomProvider`. Maybe we can move those functions to `LoomProvider`

**Comment by eduardonunesp** [comment]
Also need to organize the `loom.proto` separating `base` from `evm`, `addressmapper` and `gateway`

**Comment by eduardonunesp** [comment]
PR https://github.com/loomnetwork/loom-js/pull/199
